src/psyclone/transformations.py

```diff
@@ -237,52 +222,75 @@ def _validate(self, node1, node2):
                 "Error in LoopFuse transformation. The second loop "
                 "does not have 4 children.")
 
-        # check loop1 and loop2 have the same parent
+        # Check loop1 and loop2 have the same parent
         if not node1.sameParent(node2):
             raise TransformationError("Error in LoopFuse transformation. "
                                       "Loops do not have the same parent")
 
-        # check node1 and node2 are next to each other
+        # Check node1 and node2 are next to each other
         if abs(node1.position-node2.position) != 1:
             raise TransformationError("Error in LoopFuse transformation. "
                                       "nodes are not siblings who are "
                                       "next to each other")
-
-        # Check iteration space is the same
+        # Check that iteration space is the same
         if node1.iteration_space != node2.iteration_space:
             raise TransformationError("Error in LoopFuse transformation. "
                                       "Loops do not have the same "
                                       "iteration space")
 
     def apply(self, node1, node2):
-        '''Fuse the loops represented by :py:obj:`node1` and
-        :py:obj:`node2`.'''
+        ''' Fuses two Loops represented by `psyclone.psyGen.Node` objects
+        after performing validity checks.
+
+        :param node1: the first Node we are checking
```

Comment:
Please write in the 3rd person e.g. "the first Node that is being checked."

Reply:
Done.